components/compliance-service/habitat/hooks/run

```diff
@@ -92,7 +86,7 @@ CONFIG="$CONFIG --message-buffer-size {{cfg.service.message_buffer_size}}"
 {{~#eachAlive bind.authn-service.members as |authn|}}
 {{~#if @last}}
 addNoProxy {{authn.sys.ip}}
```

Comment:
We can likely remove a lot of these `addNoProxy` lines here and in other configuration files as well.

Reply:
ya, i've been inconsistent about it. will need to go delete those